addons/l10n_ar/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 {
     'name': 'Argentinian Accounting',
-    'version': "3.0",
+    'version': "3.1",
```

Comment:
no need to update this all the time.  It also gives fw-port problems for no reason

Reply:
Hi @jco-odoo If I'm not wrong this is needed so that odoo.sh instances update de module and get the fix / improvement. We also need this so that in our infrastructure the modules are also updated.
But if needed we can skip this change. Just let us know. Thanks!

Reply:
It is updated a bit too much this way.  But let's continue.